pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "nornir"
+version = "2.0.0"
+description = "Pluggable multi-threaded framework with inventory management to help operate collections of devices"
+authors = ["David Barroso <dbarrosop@dravetech.com>"]
+readme = "README.md"
+repository = "https://github.com/nornir-automation/nornir"
+classifiers = [
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+]
+
+[tool.poetry.dependencies]
+python = ">= 3.6, < 3.8"
+colorama = "^0.4.1"
+jinja2 = "^2.10"
+napalm = "^2.3"
+netmiko = "^2.3"
+paramiko = "^2.4"
+requests = "^2.21"
+"ruamel.yaml" = "^0.15.85"
+mypy_extensions = "^0.4.1"
+pydantic = "^0.17.0"
+[tool.poetry.dev-dependencies]
+decorator = "^4.3"
+nbval = "^0.9.1"
+pytest = "^4.1"
+pytest-cov = "^2.6"
+pylama = "^7.6"
+flake8-import-order = "^0.18.0"
+requests-mock = "^1.5"
+black = {version = "^18.3-alpha.0",allows-prereleases = true}
+mypy = "^0.650.0"
+[build-system]
```

Comment:
For all of my dev dependencies I do "*" and do `poetry update` periodically. It really helps to improve the code earlier, especially with `mypy` and `black`, which are actively developed and new features come out quite often.

Reply:
Shouldn't black be `black==18.6b4` (i.e. this is what we are currently specifying)?